src/test/runTest.ts

```diff
@@ -18,7 +19,11 @@ async function main() {
     // Passed to --extensionTestsPath
     const extensionTestsPath = path.resolve(__dirname, "./suite/index");
 
-    const vscodeExecutablePath = await downloadAndUnzipVSCode();
+    // NB: We include the exact version here instead of in `test.yml` so that
+    // we don't have to update the branch protection rules every time we bump
+    // the legacy VSCode version.
```

Comment:
Hmm, you could set the job name using the matrix, but I guess this works as well.

Alternatively we could have a job that requires test, but seems excessive for what we need.

Reply:
I'm going to ship this one as is to get CI working again, but feel free to add comments here if you have a better solution